src/screens/atomic-swap/ImportExisting.js

```diff
@@ -10,15 +10,20 @@ import Loading from "../../components/Loading";
 import React, { useEffect, useState } from "react";
 import { useHistory } from "react-router-dom";
 import { useDispatch, useSelector } from "react-redux";
-import { importProposal, proposalFetchRequested } from "../../actions";
-import { PROPOSAL_DOWNLOAD_STATUS } from "../../utils/atomicSwap";
+import { importProposal, proposalFetchRequested, proposalUpdated } from "../../actions";
+import {
+    generateReduxObjFromProposal,
+    PROPOSAL_DOWNLOAD_STATUS,
+    updatePersistentStorage
+} from "../../utils/atomicSwap";
 
 export default function ImportExisting(props) {
     // Internal state
     const [isLoading, setIsLoading] = useState(false);
     const [proposalId, setProposalId] = useState('');
     const [password, setPassword] = useState('');
     const [errorMessage, setErrorMessage] = useState('');
+    const wallet = useSelector(state => state.wallet);
```

Comment:
Why do you need it?

Reply:
I was not needed anymore, removed on [081005e](https://github.com/HathorNetwork/hathor-wallet/pull/400/commits/081005edfb48b8746d565a0361bb41382b098567)